lib/routes/admin/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+module.exports = function(mediator) {
+  const router = express.Router();
+
+  if (process.env.NODE_ENV === 'development') {
+    /**
+     * Asks for data to be reset to the default list of users
```

Comment:
I was thinking about adding `TODO:` comments to this and other parts we expect users to change out of the box so it can be picked up by editors that support it (I know at least IDEA parses it by default.)